generators/client-2/templates/src/main/webapp/app/_app.ng2module.ts

```diff
@@ -4,9 +4,7 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 // TODO change this to NgbModule -->  after complete migration
-import {NgbAlertModule} from '@ng-bootstrap/ng-bootstrap/alert/alert.module';
```

Comment:
@wmarques the issue you got with datepicker thing is from this check https://github.com/angular/angular/issues/11280